### DISCUSSION TITLE
[FEATURE REQUEST] Make jinja_[lstrip,trim]_blocks available in masterless mode

### DISCUSSION THREAD

**Discussion by joshenders** [saltstack/salt#67384] [Feature Requests]
Re-opening: https://github.com/saltstack/salt/issues/15044

Appears to have been approved in 2014 but never implemented. It would be great to have this feature. This actually lead to a production incident in our environment when somebody tried to use <%- endraw -%> incorrectly to get rid of whitespace and it ended up concatenating both lines and silently breaking a key configuration.

**Comment by welcome** [comment]
Hi there! Welcome to the Salt Community! Thank you for making your first contribution. We have a lengthy process for issues and PRs. Someone from the Core Team will follow up as soon as possible. In the meantime, here’s some information that may help as you continue your Salt journey.
Please be sure to review our [Code of Conduct](https://github.com/saltstack/salt/blob/master/CODE_OF_CONDUCT.md). Also, check out some of our community resources including:

  - [Community Wiki](https://github.com/saltstack/community/wiki)
  - [Salt’s Contributor Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html)
  - [Join our Community Slack](https://via.vmw.com/salt-slack)
  - [IRC on LiberaChat](https://web.libera.chat/#salt)
  - [Salt Project YouTube channel](https://www.youtube.com/channel/UCpveTIucFx9ljGelW63-BWg)
  - [Salt Project Twitch channel](https://www.twitch.tv/saltprojectoss)

There are lots of ways to get involved in our community. Every month, there are around a dozen opportunities to meet with other contributors and the Salt Core team and collaborate in real time. The best way to keep track is by subscribing to the Salt Community Events Calendar.
If you have additional questions, email us at saltproject@vmware.com. We’re glad you’ve joined our community and look forward to doing awesome things with you!
